Source/Scene/PointCloud.js

```diff
@@ -1080,19 +1069,26 @@ function createShaders(pointCloud, frameState, style) {
     var shaderState = {
       translucent: false,
     };
+    var functionHeader =
```

Comment:
```suggestion
    var functionSignature =
```
or maybe even `parameterList` since it doesn't include the function name.